astropy/time/formats.py

```diff
@@ -686,6 +686,82 @@ def to_value(self, timezone=None, parent=None):
     value = property(to_value)
 
 
+class TimeYMDHMS(TimeUnique):
+    """
+    ymdhms: A Time format to represent Time as year, month, day, hour,
+    minute, second (thus the name ymdhms)
+    Example::
+
+      >>> from astropy.time import Time
+      >>> t = Time({'year': [2015, 2016], 'minute': [1, 2]}, scale='utc')
+      >>> t.iso
+      array(['2015-01-01 00:01:00.000', '2016-01-01 00:02:00.000'], dtype='<U23')
+      >>> t.ymdhms.year
+      array([2015, 2016], dtype=int32)
+    """
+    name = 'ymdhms'
+
+    def _check_val_type(self, val1, val2):
+        # Note: don't care about val2 for this class
+        if not all(isinstance(val, dict) or
+                   (isinstance(val, (np.ndarray, np.record, np.void))
+                    and val.dtype.kind == 'V')
+                   for val in val1.flat):
+            raise TypeError('Input values for {0} class must be '
+                            'dict or numpy recarray or numpy structured'
+                            ' array objects'.format(self.name))
+        return val1, None
+
+    def set_jds(self, val1, val2):
+        from astropy.table import Table
```

Comment:
could you please add a comment line here why the import is not at the top of the file? (there may have been discussion about it in the previous review, but it would be more easy to know that this is by design if there is a comment about it.